src/hooks/use-photo-filter.tsx

```diff
@@ -0,0 +1,19 @@
+import { useQueryClient } from '@tanstack/react-query';
+import { Image } from 'types/unsplash-types';
+
+type UsePhotoFilterr = () => (name: string, value: string) => void;
+type ChangeFilter = (name: string, value: string) => void;
+
+export const usePhotoFilter: UsePhotoFilterr = () => {
```

Comment:
why did u add returned type here? TS will automatically define this type

Reply:
rewrited